### PR TITLE
#KTOR-936 Migrate to explicit api mode(part 6) 

### PR DESCRIPTION
**Pull request by e5l**
Fixes [`#KTOR-936`](https://youtrack.jetbrains.com/issue/KTOR-936)

### CODE REVIEW

File: ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ import java.io.*
 import java.util.concurrent.*
 
 @State(Scope.Group)
-abstract class AsyncIntegrationBenchmark<TEngine : ApplicationEngine> {
+public abstract class AsyncIntegrationBenchmark<TEngine : ApplicationEngine> {
```
Comment (cy6erGn0m) [inline ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt:26]: Can't we simply add benchmarks module to the exclusion list? This module is not used outside and looks more like a testing module? Or if we want to keep our codebase uniform, shouldn't we avoid exclusions instead?

**Review by cy6erGn0m — CHANGES_REQUESTED** [review]

File: ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt (hunk 1 above)
Comment (e5l) [reply ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt:26]: Let's exclude benchmarks. It was accidentally touched by regex.

Comment (e5l) [reply ktor-server/ktor-server-benchmarks/src/jmh/kotlin/io/ktor/server/benchmarks/AsyncIntegrationBenchmark.kt:26]: Fixed in the fixup in the last PR

**Review by Stexxe — APPROVED** [review]

**Review by cy6erGn0m — CHANGES_REQUESTED** [review]
Please add the apidump change we discussed yesterday in private. Would be nice to add a brief comment regarding why do we believe that it's safe to ignore it (just for history). The question is where should we do that: in a commit message or right here in PR.

**Review by cy6erGn0m — APPROVED** [review]

**Pull request merged**